Item/tests/Item-specs.js

```diff
@@ -56,31 +56,27 @@ describe('Item Specs', () => {
 	});
 
 	test('should support `slotBefore`', () => {
-		const expected = 'slot before';
-
 		const item = mount(
-			<Item slotBefore={expected}>
+			<Item slotBefore="slot before">
```

Comment:
why did you change this?

Reply:
Unit tests had to be adjusted in order to pass because of the change we did for `slotBefore` and `slotAfter` in computed block in `Item.js`

Reply:
I think we could use `last()` instead of `first()`.